qutip/core/cy/cqobjevo.pxd

```diff
@@ -52,6 +52,7 @@ cdef class CQobjEvo:
 
     cdef void _factor(self, double t) except *
 
+    # To remove when safe
```

Comment:
`QobjEvo` still have methods that where used for feedback support, but this will be moved to `SolverQEvo`.
There are still calls them the in `[se|me|mc]solve` function.

Reply:
I'm not sure I see why there should be a special data object for handling time evolution within solvers at all?  Even with feedback, it does seem to me that the solvers themselves should be handling it, not the data - only the solvers should know which feedback ought to be generated.

I'm not really sure why we need more data classes than `Qobj` and `QobjEvo`.  The solvers should be specifying the behaviour, rather than making a new data class each time - mixing behaviour and data too much is why the existing data classes never seem to fit quite right.  Really with the advent of `Coefficient`, even `QobjEvo` isn't completely necessary for speed - all the vdispatched time-dependence stuff is in `Coefficient` - but since we already have it there's not as much of a problem keeping it.

Reply:
Like, as far as a `Coefficient` is concerned, I wouldn't expect it to know anything about whether an argument is generated by a solver or set by a user as a constant.  Say I had a coefficient that was the cos of a frequency multiplied by an expectation value: it could be implemented something like

```python
time_dependence = qutip.coefficient('cos(expect_z * frequency * t)'
H = [qutip.sigmax(), [qutip.sigmay()]]
state = qutip.basis(2, 0)
times = np.linspace(0, 10, 101
feedbacks = [ExpectationFeedback(qutip.sigmaz(), argname='expect_z')]
qutip.sesolve(H, state, times, feedback=feedbacks)
```

Then `Coefficient` only knows about `args`, and the solver knows which feedbacks it should generate.